zkdb/src/storage-engine/ipfs.ts

```diff
@@ -254,6 +256,12 @@ export class StorageEngineIPFS extends StorageEngineBase<
         clearInterval(interval);
       }
     }, 1000);
+    console.log(
```

Comment:
We should consider to have a better logger, instead of `console.log()`

Reply:
I am not sure if we need this info for the user, but I found it useful.